internal/controller/pdbwatcher_controller.go

```diff
@@ -3,311 +3,117 @@ package controllers
 import (
 	"context"
 	"fmt"
-	"math"
-	"strconv"
-	"strings"
-	"time"
 
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
 	policyv1 "k8s.io/api/policy/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/intstr"
-	"k8s.io/client-go/tools/record"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
-
-	myappsv1 "github.com/paulgmiller/k8s-pdb-autoscaler/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-// PDBWatcherReconciler reconciles a PDBWatcher object
-type PDBWatcherReconciler struct {
-	client.Client
-	Scheme   *runtime.Scheme
-	Recorder record.EventRecorder
+// PDBWatcher is a custom resource representing a PDBWatcher.
+type PDBWatcher struct {
```

Comment:
this should come out of api/v1 where kubebuilder generated it. Trying not to move too far from kube-builder layout as they often have reasons fro the layout